src/bundle/Resources/public/scss/ui/modules/universal-discovery/_content.create.button.scss

```diff
@@ -1,10 +1,14 @@
 .c-content-create-button {
     margin-right: calculateRem(10px);
-
-    &__btn {
+    
+    & &__btn {
+        margin-right: calculateRem(52px);
         padding: calculateRem(4px);
-        width: calculateRem(34px);
-        height: calculateRem(34px);
-        display: flex;
+        color: $ibexa-color-primary;
+        text-decoration: none;
+    }
+
+    .ez-icon {
+        fill: $ibexa-color-primary;
```

Comment:
I think the primary color deserves to have an icon modifier.

Reply:
We have modifier `ez-icon--primary`